source/features/rgh-improve-new-issue-form.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import clearCacheHandler from '../helpers/clear-cache-handler';
+import {isRefinedGitHubRepo} from '../github-helpers';
+
+function init(): false | void {
+	if (new URL(location.href).searchParams.get('template') !== '1_bug_report.yml') {
+		return false;
+	}
+
+	const {version} = browser.runtime.getManifest();
+	select('input#issue_form_extension_version')!.value = version;
```

Comment:
We probably want to remove this line:

https://github.com/refined-github/refined-github/blob/6c6e2799cd1499f6e629cf9764e0e23b3fd19ce5/source/features/index.tsx#L73

Reply:
We could keep it, in case the user disables the extension in between the time they see the error and open the link. (It's pretty unlikely though.)